### PR TITLE
Refactor: consistent naming and more details on retuend object

### PR DESCRIPTION
**Pull request by pawelgrzybek**

### CODE REVIEW

File: aws/aws/lib/lambdas/syncBlockProd.mjs [hunk 1]
```diff
@@ -62,8 +62,10 @@ exports.handler = async event => {
           historicalData: filteredData.slice(0, Math.min(MAX_L2_BLOCKS, filteredData.length)),
           maxBlock,
           isLast: !dataBlockNext?.Item,
-          hashBlockLast: dataBlockLast?.Item?.blockHash ?? null,
-          hashBlockNext: dataBlockNext?.Item?.blockHash ?? null,
+          rootBlockLast: dataBlockLast?.Item?.block?.root ?? null,
```
Comment (daveroga) [inline aws/aws/lib/lambdas/syncBlockProd.mjs:65]: Why we change from `hashBlockLast` and `hashBlockNext` to `rootBlockLast` and `rootBlockNext`?
Is not enough by keeping the previous version with the block hash and adding `numberBlockLast` and `numberBlockNext`?

Comment (pawelgrzybek) [reply aws/aws/lib/lambdas/syncBlockProd.mjs:65]: Hi David. Good question. Unfortunately the information about `hash` by itself is not enough to compare two block in wallet. We store only `root` in `timber` table that allows me to associate `timber` with the `block`. Thats why I need this info from the backend.

**Review by daveroga — APPROVED** [review]

**Review by druiz0992 — CHANGES_REQUESTED** [review]
Just add modified wallet files required from ´nightfall_3` to tmp folder

File: aws/Makefile [hunk 2]
```diff
@@ -278,6 +278,8 @@ deploy-wallet:
 		cp tmp/wallet/TokenType.ts nightfall_3/wallet/src/static/supported-token-lists; \
 		cp tmp/wallet/walletIndex.jsx nightfall_3/wallet/src/views/wallet/index.jsx; \
 		cp tmp/wallet/local-storage.js nightfall_3/wallet/src/utils/lib/local-storage.js; \
+		cp tmp/wallet/index.jsx nightfall_3/wallet/src/hooks/User/index.jsx; \
```
Comment (druiz0992) [inline aws/Makefile:281]: changes also required in `deploy-wallet-local`

**Review by druiz0992 — APPROVED** [review]

**Pull request merged**